package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "habitica",
-  "version": "2.2.1",
+  "version": "0.0.0",
```

Comment:
Just to be sure I understand, this is being set to 0.0.0 because once this PR is merged and in use, Zapier will be reading the version number from the GitHub tag. Zapier will be ignoring the version in this file, right?

Reply:
I sat it to 0.0.0 to avoid confusion that might be caused by any other version. Yes `npm version $(git describe --tags --abbrev=0)` will look at the most recent tag and set it to that.  If you see a value in keeping this updated, I believe it would be possible to automate that too but I dislike cluttering the history of a repo with automated commits. It's a preference thing :p .

Reply:
@Majed6 Thanks! That sounds great just as it is. I'm happy to have it stay as 0.0.0 in this file.